aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -415,6 +517,28 @@ void gemm(
     return;
   }
 #endif
+
+#if AT_BUILD_WITH_BLAS()
+  if (use_blas_gemm(transa, transb, m, n, k, lda, ldb, ldc)) {
+    int m_ = m, n_ = n, k_ = k, lda_ = lda, ldb_ = ldb, ldc_ = ldc;
+    float alpha_ = alpha, beta_ = beta;
+    int a_size = (transa != TransposeType::NoTranspose?m_:k_) * lda;
+    std::vector<float> float_a(a, a + a_size);
+    int b_size = (transb != TransposeType::NoTranspose?k_:n_) * ldb;
```

Comment:
looks like @malfet changed the size computation in other places but not here; which is correct? probably explains our segfault, right?

Reply:
I think the right think might just be a_size = m * k, b_size = k * n, c_size = m * n? no messing around with lda/ldb/ldc